app/controllers/admin_controller.rb

```diff
@@ -48,4 +53,17 @@ def check_authorisation
       redirect_to root_path
     end
   end
+
+  def support_payload
+    {
+      subject: "Specialist Finder Edit Request: #{current_format.title}",
+      tags: %w[specialist_finder_edit_request],
```

Comment:
Awesome! For reference, I've just raised a [ticket with User Support](https://govuk.zendesk.com/agent/tickets/5957502) to set up the trigger rule that will route tickets with this tag, into the Publishing Service Support queue.